README.md

```diff
@@ -76,9 +78,51 @@ In some environments, the CLI is able to infer parameters like build or job
 numbers and commit hashes for you. In this case, you can omit the respective
 paramters. These systems are:
 
-* Travis CI
-* AppVeyor
-* Buildkite
+- Travis CI
+- AppVeyor
+- Buildkite
+
+### Updating Jobs
+
+Create new or update existing jobs in Zeus, providing a job ID and the
+corresponding build ID. If the provided build does not exist, it will be
+created. On supported CI systems, the job and build id are automatically
+inferred from the environment.
+
+**Note:** if the corresponding build does not exist, you must provide `--ref`
+attribute, otherwise Zeus will fail to create the build.
+
+```text
+zeus job update
+
+Add/update a job
+
+Options:
+  --url, -u            Custom URL                                       [string]
+  --token              Token for authorized access to Zeus              [string]
+  --job, -j            Job id                                [number] [required]
+  --build, -b          Build id                              [number] [required]
+  --ref, -r            Commit hash                                      [string]
+  --build-label, --bl  Custom build label                               [string]
+  --job-label, --jl    Custom job label                                 [string]
+  --status, -s         Job execution status
+                               [string] [choices: "pending", "passed", "failed"]
+  -v, --version        Show version number                             [boolean]
+  -h, --help           Show help                                       [boolean]
+```
+
+**Examples:**
+
+```sh
+# Create a new job in build number 234
+zeus job update --number=234 --build=123
```

Comment:
Thinking about it, we should give an example where we use an environment variable to make it clear to people that they should not make up stuff here and also add the ref argument so that people don't forget it, since it's effectively required: `zeus job update -b "$TRAVIS_BUILD_ID" -j "$TRAVIS_JOB_ID" -r "$TRAVIS_COMMIT"`